doc/engineering-reference/src/integrated-solution-manager/zone-air-mass-flow-conservation.tex

```diff
@@ -166,6 +194,32 @@ \subsection{Zone Mixing Flow Rate Calculations}\label{zone-mixing-flow-rate-calc
 {\dot m_{XR,\,new,j}} = \left( {{{\dot m}_{XR,jDesign}}/{{\dot m}_{XR,Design}}} \right) \cdot {\dot m_{XR,\,new}}
 \end{equation}
 
+\textbf{AdjustReturnOnly:} adjusts the zone total return air mass flow rate only to enforce zone air mass flow balance while the zone mixing object mass flow rate are kept at user specified values. Infiltration air flow can also adjusted if required depending on user preference as specified in Section \textit{Infiltration Flow rates Adjustments}.
+
+First, determine the user specified mixing object received and source air flow rates of the current zone, then determine the zone total return air flow rate as follows:
+
+\begin{equation}
+{\dot m_{XS}} = \sum\nolimits_j {{{\dot m}_{XS,j}}}
+\end{equation}
+
+\begin{equation}
+{\dot m_{XR}} = \sum\nolimits_j {{{\dot m}_{XR,j}}}
+\end{equation}
+
+If ZoneAirMassFlowConservation is active, then determine the total zone return air flow rate:
+\begin{equation}
+{\dot m_{R}} = MAX\left( {0.0,\,{{\dot m}_S} - {{\dot m}_{EX,tot}} + [{{\dot m}_{XR}} - {{\dot m}_{XS}}]} \right)
+\end{equation}
+
+Also checks that the zone total return air mass flow rate does not exceed the airloop design supply flow rate as follows:
+\begin{equation}
+{\dot m_{R}} = MIN\left( {{\dot m_{R}},\,{{\dot m}_{DesSupply,i}}} \right) 
+\end{equation}
+
+\textbf{AdjustMixingThenReturn:} first adjusts the zone mixing air mass flow rates, then adjusts the zone total return air mass flow rate to enforce zone air mass flow balance. Infiltration air flow can also adjusted if required depending on user preference as specified in section \textit{Infiltration Flow rates Adjustments}. For adjusting the mixing mass flow rates the set of equations for \textit{AdjustMixingOnly} method described above are used and for adjusting the zone total return air mass flow rate the equation for \textit{AdjustReturnOnly} method is used. 
+
+\textbf{AdjustReturnThenMixing:} first adjusts the zone total return air mass flow rate, then adjusts the zone mixing mass flow rates to enforce zone air mass flow balance. Infiltration air flow can also adjusted if required depending on user preference as specified in section \textit{Infiltration Flow rates Adjustments}. For adjusting the zone total return air mass flow rates the equations for \textit{AdjustReturnOnly} method described above is used and for adjusting the zone mixing mass flow rates the set of equations for \textit{AdjustMixingOnly} method are used.
+
```

Comment:
Updated the engineering reference to reflect the changes as applicable for each of the zone air flow conservation enforcing methods.